xml/operations-maintenance-deploy-ptf.xml

```diff
@@ -182,5 +175,47 @@ New venv installed
 drwxr-xr-x 10 root root 4096 May 28 09:30 nova-20180528T092954Z
    drwxr-xr-x 10 root root 4096 Aug 14 16:13 nova-20180814T161306Z</screen>
   </step>
+  <step>
+   <para>
+    The PTF may also have <literal>RPM</literal> package updates in addition to
+    venv updates. ardana-update-pkgs.yml -
```

Comment:
Is this `ardana-update-pkgs.yml -` supposed to be floating? Or perhaps marked up?